packages/react-ai/src/components/AIConversation/context/ControlsContext.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { ConversationInputContext } from './ConversationInputContext';
+import { SuggestedPrompt } from '../types';
+import { ConversationMessage } from '../../../types';
+
+export interface ControlsContextProps {
+  Form?: React.ComponentType<
+    {
+      handleSubmit: (e: React.FormEvent<HTMLFormElement>) => void;
+    } & Required<ConversationInputContext>
+  >;
```

Comment:
Is this an exposed component for override/composition? Usage of required props complicates DX/non-idiomatic to the composable API

Reply:
It is exposed, maybe we can sync with @dbanksdesign on this